src/Repository/ProductRepository.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Repository;
+
+class ProductRepository {
+
+    private array $products;
+
+    public function __construct() {
+        $db = file_get_contents(dirname(__DIR__, 2) . '/database/data.json');
+
+        $this->products = json_decode($db, true);
+    }
+
+    public function getList() {
+        return $this->products;
+    }
+
+    public function getProduct(int $id) {
+        return $this->products[$id - 1];
```

Comment:
maybe you should explicitly define keys in this array -> it would be more robust than just expecting this array to be sorted properly